### PR TITLE
Mobile finish

### PR DESCRIPTION
**Pull request by brightbrain20**
- created work section and about my self section.
- check for linter errors.
- stick to figma design.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi  @brightbrain20 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟥 

You have submitted a project, that is INVALID because it does neither follows the github flow or git-flow. Your PR should be made from the “feature branch” into the `main` branch. Please change your base branch `master`  to `main` using this   [Changing the base branch of a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -44,6 +44,194 @@ <h1>Hey There. I'm Bright</h1>
...
+        <h2>About me</h2>
+        <p class="footer-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#8" class="footer-btm button">Get My Resume</a>
+      </div>
+      <div class="footer-img">
+        <img src="/Images/illustration-about-me-mobile.png" alt="">
```
Comment (Ibizugbe) [inline index.html]: You have done a fantastic job so far😉😉👍 Just this issue🥲 Kindly ensure an alt attribute is always added, and it shouldn't be left empty😉 this makes your project accessible to screen readers and it is also a best practice in html🤗🤗 Kindly make the necessary change🤗🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @brightbrain20 👋👋, 

198009982-d99c2507-f20d-4c00-90a3-a54097598b36


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are properly setup and they are passing👍
- [x] Nice work sticking to the design👍
- [x] Github flow practice has been used👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @brightbrain20   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



## Highlights
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**